install/common-pre-install/hpcm_installation-cpi.md

```diff
@@ -699,21 +699,21 @@ See the [set boot order](../../background/ncn_boot_workflow.md#setting-boot-orde
 
                ```bash
                cm node discover status | grep client-hostname  > temp.txt
-               for i in `cat temp.txt| awk '{print $1}'` ; do echo $i; ipmitool -I lanplus -H $i -U root -P initial0 power status; done
+               for i in `cat temp.txt| awk '{print $1}'` ; do echo $i; ipmitool -I lanplus -H $i -U [USERNAME] -P [PASSWORD] power status; done
                ```
 
             1. Check if all the node's bootorder is set to `pxe`.
 
                ```bash
-               for i in `cat temp.txt| awk '{print $1}'` ; do echo $i; ipmitool -I lanplus -H $i -U root -P initial0 chassis bootdev; done
+               for i in `cat temp.txt| awk '{print $1}'` ; do echo $i; ipmitool -I lanplus -H $i -U [USERNAME] -P [PASSWORD] chassis bootdev; done
                ```
 
             1. Set the boot order to `pxe` and reboot the nodes.
 
                ```bash
-               for i in `cat temp.txt| awk '{print $1}'` ; do echo $i; ipmitool -I lanplus -H $i -U root -P initial0 chassis bootdev pxe; done
-               for i in `cat temp.txt| awk '{print $1}'` ; do echo $i; ipmitool -I lanplus -H $i -U root -P initial0 power on; done
-               for i in `cat temp.txt| awk '{print $1}'` ; do echo $i; ipmitool -I lanplus -H $i -U root -P initial0 power reset; done
+               for i in `cat temp.txt| awk '{print $1}'` ; do echo $i; ipmitool -I lanplus -H $i -U [USERNAME] -P [PASSWORD] chassis bootdev pxe; done
```

Comment:
Use a variable for `USERNAME` and `PASSWORD`, otherwise the format of these commands is not copy-paste friendly

`ipmitool` actually reads the password from a variable named `IPMI_PASSWORD` if given the `-E` argument.
```
IPMI_PASSWORD=foo
ipmitool -E
```

Please set and use
- `USERNAME`
- `IPMI_PASSWORD`

Reply:
Updated the IPMI commands